From: olh@suse.de
Subject: not having 0xaa55 in the ROM is not an error

Quiet an incorrect warning in aty128fb and radeonfb about the PCI ROM
content. Macs work just find without that signature.


Signed-off-by: Olaf Hering <olh@suse.de>

 drivers/video/aty/aty128fb.c    |    2 +-
 drivers/video/aty/radeon_base.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

Index: linux-2.6.13-fb-rom/drivers/video/aty/aty128fb.c
===================================================================
--- linux-2.6.13-fb-rom.orig/drivers/video/aty/aty128fb.c
+++ linux-2.6.13-fb-rom/drivers/video/aty/aty128fb.c
@@ -806,7 +806,7 @@ static void __iomem * __init aty128_map_
 
 	/* Very simple test to make sure it appeared */
 	if (BIOS_IN16(0) != 0xaa55) {
-		printk(KERN_ERR "aty128fb: Invalid ROM signature %x should be 0xaa55\n",
+		printk(KERN_DEBUG "aty128fb: Invalid ROM signature %x should be 0xaa55\n",
 		       BIOS_IN16(0));
 		goto failed;
 	}
Index: linux-2.6.13-fb-rom/drivers/video/aty/radeon_base.c
===================================================================
--- linux-2.6.13-fb-rom.orig/drivers/video/aty/radeon_base.c
+++ linux-2.6.13-fb-rom/drivers/video/aty/radeon_base.c
@@ -329,7 +329,7 @@ static int __devinit radeon_map_ROM(stru
 
 	/* Very simple test to make sure it appeared */
 	if (BIOS_IN16(0) != 0xaa55) {
-		printk(KERN_ERR "radeonfb (%s): Invalid ROM signature %x should be"
+		printk(KERN_DEBUG "radeonfb (%s): Invalid ROM signature %x should be"
 		       "0xaa55\n", pci_name(rinfo->pdev), BIOS_IN16(0));
 		goto failed;
 	}
